src/ui/src/js/projects.js

```diff
@@ -23,6 +23,8 @@ window.$ = window.jQuery = jQuery;
 
 import "jstree";
 
+var dt = require("datatables.net")(window, $);
```

Comment:
```suggestion
import dt from "datatables.net";
dt(window, $);
```

So we use imports all over. 😉